flashlight/fl/nn/modules/BatchNorm.cpp

```diff
@@ -44,6 +44,21 @@ BatchNorm::BatchNorm(
   initialize();
 }
 
+BatchNorm::BatchNorm(const BatchNorm& other)
+    : featAxis_(other.featAxis_),
+      featSize_(other.featSize_),
+      numBatchesTracked_(other.numBatchesTracked_),
+      runningMean_(
+          other.runningMean_.tensor().copy(),
```

Comment:
looks good for now. We can add `Variable::copy` which wraps this down the line.

Reply:
Sounds good.